app/admin/allocations.rb

```diff
@@ -87,9 +87,13 @@
                                       .new(current_user.company)
                                       .not_allocated_including(f.object.user)
                                       .select(:id, :name)
-
+      
         input :user, as: :select, collection: company_users_not_allocated
-        input :project, collection: current_user.company.projects.active.order(:name)
+        if params[:action] == 'edit'
+          input :project, collection: current_user.company.projects.order(:name)
```

Comment:
why?

Reply:
So you can't create a new allocation to a inactive project